plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -2,7 +2,7 @@
     {% include visualizationTemplate %}
 {%- else -%}
 
-{% set isDataTableEmpty = (dataTable is empty or dataTableHasNoData|default(false)) %}
+{% set isDataTableEmpty = (dataTable is not defined or dataTableHasNoData|default(false)) %}
```

Comment:
might it be possible that `dataTable` is `null` in some cases? If so it might be better to maybe also check for `dataTable is null`

Reply:
Done 👍